### PR TITLE
add links managed in admin

### PR DESCRIPTION
**Pull request by sonlinux**
fix kartoza/LEDET_BIMS#53

demo_final

### CODE REVIEW

**Comment by dimasciput** [comment]
@sonlinux please put required library in https://github.com/kartoza/django-bims/blob/develop/setup.py, so it will be installed other projects.

**Comment by ann26** [comment]
Hi @sonlinux can you add screenshots please?

File: bims/views/links.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+
+class LinksPageView(TemplateView):
+    """View to show the links page."""
+    template_name = 'links.html'
+
+    def get_context_data(self, **kwargs):
+
+        context = super(LinksPageView, self).get_context_data(**kwargs)
+        context['Titile'] = 'Links Page'
```
Comment (ann26) [inline bims/views/links.py]: what is this for?

File: bims/views/links.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+        for category in categories:
+            category_links = {'category': category, 'links': []}
+
+            links_qs = Link.objects.filter(category=category.pk)
+            for link in links_qs:
+                category_links['links'].append(link)
+
+            context['data_list'].append(category_links)
+            context['category_links'] = category_links
+            context['categoriez'] = categories
```
Comment (ann26) [inline bims/views/links.py]: I think the context name should be `categories`

**Comment by timlinux** [comment]
@sonlinux lets get moving with this ... please address the comments above, make sure your work is exposed to end-users too and get this work mergable then merged when review passes.

**Comment by sonlinux** [comment]
Thanks,
I have it done in the LEDET base repository

**Comment by dimasciput** [comment]
@sonlinux 
I added a link, but it doesn't shows up in links page

screen shot 2018-06-18 at 12 32 49 pm

Also don't create a new app, put in the bims app, otherwise it won't shows up in LEDET.

**Comment by sonlinux** [comment]
# Its working @dimasciput

ccbv

**Comment by dimasciput** [comment]
LGTM

**Pull request merged**